### PR TITLE
Review of install.md

### PR DESCRIPTION
**Pull request by sabeena21**
### Change Description ###

Made minor modifications in the content.

### CODE REVIEW

File: docs/application/tizen-studio/rt-ide/getting-started/install.md [hunk 1]
```diff
@@ -4,43 +4,41 @@ To install the Tizen Studio for RT:
...
 
-   The license contains important legal notices for using the Tizen Studio. Read it fully, and click **Accept** only if you agree with the license statement.
+   The license contains important legal notices for using the Tizen Studio. Read the complete agreement, and click **Accept** only if you agree with the license statement:
 
    ![Software license agreement](./media/rt_install_license.png)
 
 2. Configure the installation directory.
 
-   The Tizen Studio for RT and its data directory are installed in the location specified in the Installation Settings step. To change the directory, click the **...** button and specify a new directory. If the new directory is valid, click **Install** at the bottom of the installer.
+   The Tizen Studio for RT and its data directory are installed in the location specified in the Installation Settings step. To change the directory, click ![Button](./media/rt_button.png) and specify a new directory. If the new directory is valid, click **Install**:
```
[the remark this answers is not in the document]
Comment (sabeena21) [reply docs/application/tizen-studio/rt-ide/getting-started/install.md]: @journey2w , Okay, thank you so much for guiding me on this. I will replace the image with text word.

**Comment by huytaduc** [comment]
checkout the webpage https://docs.stg.tizen.org/staging/486

**Comment by huytaduc** [comment]
checkout the webpage https://docs.stg.tizen.org/staging/486

**Comment by mijin80lee** [comment]
TD Sync Completed.(reviewing state)

**Pull request merged**